concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -109,181 +103,147 @@ def report_computed_task(timestamp = None, task_to_compute = None):
 
 
 def main():
-    cluster_url     = parse_command_line(sys.argv)
-    current_time    = get_current_utc_timestamp()
-    cluster_consts  = get_protocol_constants(cluster_url)
-    #  Test CASE 2A + 2D + 3 - Send ForceSubtaskResults with same task_id as stored by Concent before
-    #  Step 1. Send ForceSubtaskResults first time
-    subtask_id = str(random.randrange(1, 100000))
-    task_id = subtask_id + '2A'
+    (cluster_url, patterns) = parse_arguments()
+    cluster_consts = get_protocol_constants(cluster_url)
+    print_protocol_constants(cluster_consts)
+
+    test_id = str(random.randrange(1, 100000))
+    tests_to_execute = get_tests_list(patterns, list(globals().keys()))
```

Comment:
As discussed before, please don't use `globals()`. What you are trying to do here is implement your own test runner, something which is already available if you use python default `unittest` or nose tests by rewritting test functions into test class.

As discussed, we don't want to change it now, but we should in the future.